ScheduleAPI/Models/ScheduleRepeatDaysOfWeekFlags.cs

```diff
@@ -11,6 +11,8 @@ public enum ScheduleRepeatDaysOfWeekFlags
         Wednesday = 8,
         Thursday = 16,
         Friday = 32,
-        Saturday = 64
+        Saturday = 64,
+        Weekdays = Monday | Tuesday | Wednesday | Thursday | Friday,
+        Everyday = Weekdays | Saturday | Sunday
```

Comment:
Thanks for adding these, really cleans up the tests and communicates intent through better use of language.

Reply:
:)